validator/etc/log_config.toml.example

```diff
@@ -0,0 +1,29 @@
+version = 1
+disable_existing_loggers = false
+[formatters.simple]
```

Comment:
can toml tolerate linefeeds before the section headings (in brackets)?  This is a bit hard to read all crammed together.